meson.build

```diff
@@ -0,0 +1,67 @@
+project('loguru', 'cpp',
```

Comment:
Could you please use `emilk-loguru` for consistency?

Reply:
Done.